source/common/nats/streaming/BUILD

```diff
@@ -49,6 +49,7 @@ envoy_cc_library(
     hdrs = ["connect_response_handler.h"],
     repository = "@envoy",
     deps = [
+        "@envoy//envoy/common:pure_lib",
```

Comment:
Why is this change necessary? I can see that it was introduced with https://github.com/solo-io/envoy-gloo/commit/026e3cfdb3647d4d83c9a34fb5c7529d324c8c72 , but is this addition necessary and if so what does it do?

Reply:
Upstream sharded out some of its sub libraries so its no longer part of the base included stuff https://github.com/envoyproxy/envoy/commit/1a37802c9c8b2a163305774bbbf2ba21318531d2